pkg/dumper/mysql/dumper.go

```diff
@@ -51,7 +51,11 @@ func (p *myDumper) DumpTable(tableName string, rowChan <-chan database.Row) erro
 	if err != nil {
 		return errors.Wrap(err, "failed to insert rows")
 	}
-	log.WithField("table", tableName).WithField("inserted", insertedRows).Debug("inserted rows")
+
+	log.WithFields(log.Fields{
+		"table", tableName,
```

Comment:
How can this work? This should be a map.

Reply:
Go to 🛌 vlad now